src/Chainweb/Pact/Backend/Bench.hs

```diff
@@ -51,12 +60,37 @@ bench = C.bgroup "pact-backend" $
       where
         playOne n = map ($ n) benches
 
-cpBenchNoRewindOverBlock :: Int -> C.Benchmark
-cpBenchNoRewindOverBlock transactionCount =
+pactSqliteWithBench :: Bool -> (PactDbEnv (DbEnv PSQL.SQLite) -> C.Benchmark) -> C.Benchmark
+pactSqliteWithBench unsafe benchtorun = C.envWithCleanup setup teardown $ \ ~(NoopNFData (e,_)) -> C.bgroup tname (benches e)
```

Comment:
(Broken record incoming)... please conform to style guide:

  - 80 column length limit
  - four column indent after `do`
  - indent toplevels that continue lines like this:

```
someLongFooName
    :: Bar
    -> IO Baz
    -> IO Quux
```